source/material_model/reaction_model/katz2003_mantle_melting.cc

```diff
@@ -261,6 +582,42 @@ namespace aspect
         M_cpx = prm.get_double ("Mass fraction cpx");
         peridotite_melting_entropy_change
           = prm.get_double ("Peridotite melting entropy change");
+
+        reference_rho_fluid         = prm.get_double ("Reference melt density");
+        xi_0                       = prm.get_double ("Reference bulk viscosity");
+        viscosity_fluid                      = prm.get_double ("Reference melt viscosity");
```

Comment:
This is really nitpicky, but it looks nicer if you change it to 
```suggestion
        reference_rho_fluid        = prm.get_double ("Reference melt density");
        xi_0                       = prm.get_double ("Reference bulk viscosity");
        viscosity_fluid            = prm.get_double ("Reference melt viscosity");
```